src/test/rgw/test_d4n_policy.cc

```diff
@@ -169,7 +169,7 @@ TEST_F(LFUDAPolicyFixture, LocalGetBlockYield)
     dynamic_cast<rgw::d4n::LFUDAPolicy*>(policyDriver->get_cache_policy())->save_y(optional_yield{yield});
     policyDriver->get_cache_policy()->init(env->cct, env->dpp, io, driver);
 
-    std::string key = block->cacheObj.bucketName + "_" + block->cacheObj.objName + "_" + std::to_string(block->blockID) + "_" + std::to_string(block->size);
+    std::string key = block->cacheObj.bucketName + "#" + block->cacheObj.objName + "#" + std::to_string(block->blockID) + "#" + std::to_string(block->size);
```

Comment:
please decouple these changes from dirty prefix removal so that i can squash the commit with the correct one.

Reply:
ok it is a separate commit, ignore my comment. but update the description of the PR - it is misleading

Reply:
And please also modify the commit message to be more descriptive